db/thrman.c

```diff
@@ -51,6 +51,8 @@
 #include "reqlog.h"
 #include "str0.h"
 
+#include <unistd.h>
```

Comment:
leftover?

Reply:
Yes, I had a sleep in there earlier to try to induce a race.